src/art/anshuman3/script.js

```diff
@@ -0,0 +1,24 @@
+const charcter = document.getElementById('character');
+const charcontainer = charcter.getContext('2d');
+const charwid = charcter.width = 623;
+const charhit = charcter.height = 572;
+const charimg = new Image();
+charimg.src = "sprite.png";
```

Comment:
Please provide the license to the image you used, so that we can be sure that we don't violate copyright.